packages/geolocation/src/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const welcome = require("./utils/welcomePackage")
+
+module.exports = {
+    geolocationWelcome: welcome,
+}
```

Comment:
The linter had left you this message:  “No newline at end of file”, make sure to follow this kind of recommendation.

Reply:
ready I already add the line break